lib/v2/bilibili/liveSearch.js

```diff
@@ -15,26 +17,33 @@ module.exports = async (ctx) => {
             orderTitle = '人气直播';
             break;
     }
+    const verifyString = await cache.getVerifyString(ctx);
+    let params = `__refresh__=true&_extra=&context=&page=1&page_size=42&order=online&duration=&from_source=&from_spmid=333.337&platform=pc&highlight=1&single_column=0&keyword=${urlEncodedKey}&qv_id=68nr2HBn3S27waWVjvFT0CLT7U3TF0U3&ad_resource=&source_tag=3&gaia_vtoken=&category_id=&search_type=live&dynamic_offset=0&web_location=1430654`;
```

Comment:
- `order` is fixed to `online` in your `params` which means users will always get the same result no matter if they pass `live_time` or `online` to `ctx.params.order`.
- You can remove `qv_id` from params since it's a random generated string.

<details><summary>For those who are wondering how <code>qv_id</code> is generated</summary>

```js
var ID_CHAR_SET = {
    number: "0123456789",
    letter: "abcdefghijklmnopqrstuvwxyz",
    special: "~`!@#$%^&*()-_+=[]{};:\"',<.>/?"
}
var charNb = ID_CHAR_SET.number
  , charLower = ID_CHAR_SET.letter
  , charUpper = charLower.toUpperCase()
  , charSpecial = ID_CHAR_SET.special;
function isArr(st) {
    return st instanceof Array
}
function isBol(st) {
    return typeof st == "boolean"
}
function isNum$3(st, dt) {
    var Et = typeof st == "number";
    return dt ? Et : Et && !Number.isNaN(st) && Number.isFinite(st)
}
function isStr$2(st) {
    return typeof st == "string"
}
function isUdf(st) {
    return st === void 0
}
function randomNum(st, dt, Et) {
    var St = Math.random();
    if (!isNum$3(st))
        return St;
    isBol(dt) && (Et = dt);
    var wt;
    return !isNum$3(dt) || dt === st ? wt = St * st : wt = St * (dt - st) + st,
    Et ? wt : Math.floor(wt)
}
function unique(st, dt) {
    var Et = isStr$2(st);
    Et && (st = st.split("")),
    isArr(st) || (st = []);
    for (var St = st.slice(), wt = -1; ++wt < St.length; )
        for (var xt = wt, kt = function() {
            var Wt = St[wt]
              , zt = St[xt]
              , Xt = isArr(dt)
              , Qt = !Xt && (dt ? Wt == zt : Wt === zt) || Xt && dt.some(function(Zt) {
                return Wt && zt && !isUdf(Wt[Zt]) && Wt[Zt] === zt[Zt]
            });
            Qt && St.splice(xt--, 1)
        }; ++xt < St.length; )
            kt();
    return Et ? St.join("") : St
}

function randomId(st, dt) {
    isStr$2(st) && (dt = st),
    isNum$3(st) || (st = 12);
    var Et = "";
    !dt || !isStr$2(dt) ? Et = charNb + charUpper : dt === "[*]" ? Et = charNb + charLower + charUpper + charSpecial : dt.match(/0-9|a-z|A-Z|\[s\]/) ? (/0-9/.test(dt) && (Et += charNb),
    /a-z/.test(dt) && (Et += charLower),
    /A-Z/.test(dt) && (Et += charUpper),
    /\[s\]/.test(dt) && (Et += charSpecial),
    Et += unique(dt.replace(/0-9|a-z|A-Z|\[s\]/g, ""))) : Et = dt;
    for (var St = "", wt = -1; ++wt < st; )
        St += Et[randomNum(Et.length)];
    return St
}

// qv_id: Re.value
// Re.value = Aa(32, "0-9a-zA-Z")
const qv_id = randomId(32, '0-9a-zA-Z')
```
</details>